clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -750,54 +750,92 @@ __dpct_inline__ void load_striped(const Item &item, InputIteratorT block_itr,
 /// Load a linear segment of elements into a blocked arrangement across the
 /// work-group.
 ///
-/// \tparam InputT The data type to load.
-///
+/// \tparam T The data type to load.
 /// \tparam ElementsPerWorkItem The number of consecutive elements partitioned
 /// onto each work-item.
-///
 /// \tparam InputIteratorT  The random-access iterator type for input \iterator.
-///
-/// \param linear_tid A suitable linear identifier for the calling work-item.
-///
-/// \param block_itr The work-group's base input iterator for loading from.
-///
-/// \param items Data to load
-template <typename InputT, size_t ElementsPerWorkItem, typename InputIteratorT>
-__dpct_inline__ void load_direct_blocked(size_t linear_tid,
-                                         InputIteratorT block_itr,
-                                         InputT (&items)[ElementsPerWorkItem]) {
+/// \param item The calling work-item.
+/// \param input_iter The work-group's base input iterator for loading from.
+/// \param data Data to load.
+template <typename T, size_t ElementsPerWorkItem, typename InputIteratorT,
+          typename ItemT>
+__dpct_inline__ void load_direct_blocked(const ItemT &item,
```

Comment:
can we change the API signature in header file?

Reply:
The API signature not changed